TestCases/compliance-level-3/0081-feel-getentries-function/0081-feel-getentries-function-test-01.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Contributed to DMN TCK by StrayAlien -->
+<testCases xmlns="http://www.omg.org/spec/DMN/20160719/testcase" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation=""  xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <modelName>0081-feel-getentries-function.dmn</modelName>
+    <labels>
+        <label>Compliance Level 3</label>
+        <label>FEEL Functions: map</label>
+    </labels>
+
+    <testCase id="decision_001">
+        <description>no param</description>
+        <resultNode errorResult="true" name="decision_001" type="decision">
+            <expected>
+                <value xsi:nil="true"></value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="decision_002">
+        <description>two many params</description>
+        <resultNode errorResult="true" name="decision_002" type="decision">
+            <expected>
+                <value xsi:nil="true"></value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="decision_003">
+        <description>null params</description>
+        <resultNode errorResult="true" name="decision_003" type="decision">
+            <expected>
+                <value xsi:nil="true"></value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="decision_004">
+        <description>get a value</description>
+        <resultNode name="decision_004" type="decision">
+            <expected>
+                <list>
+                    <item>
+                        <component name="a">
+                            <value xsi:type="xsd:string">foo</value>
+                        </component>
+                    </item>
+                    <item>
+                        <component name="b">
+                            <value xsi:type="xsd:string">bar</value>
```

Comment:
I disagree.
the DMNv1.2 spec says:
![image](https://user-images.githubusercontent.com/1699252/51471009-caa99000-1d75-11e9-9849-7376112aa5f7.png)

based on the example, the correct result of 
```
get entries({a: "foo", b: "bar"})
```

must be
```
[{value=foo, key=a}, {value=bar, key=b}]
```

In order to satisfy the example provided by the DMNv1.2 spec itself.

Reply:
Thanks @tarilabs.  Great catch.  You are correct.  I'll amend.

Reply:
@tarilabs. test updated as such.